src/utils/http.ts

```diff
@@ -97,7 +97,7 @@ export class Http {
       apiError = new ApiError(
         error.message,
         error.response.status,
-        error.response.data
+        error.response.data as string
```

Comment:
Could you add some info into the commit message as to why we need this change? Is this an axios bump thing?

Reply:
There was a failure in the pre-merge checks regarding of error on that line:

`Argument of type 'unknown' is not assignable to parameter of type 'string'.
`
Only seems to come up when axios is upgraded. 

I've now bumped the version to 1.6.8, and that error no longer exists